Traceback (most recent call last):
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/LOGistICS.py", line 30, in <module>
    import template_selector as ts
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/template_selector.py", line 5, in <module>
    df = pd.read_csv('template.csv', delimiter=',',header=0)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 317, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1729, in _make_engine
    self.handles = get_handle(
  File "/usr/lib/python3.10/site-packages/pandas/io/common.py", line 857, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'template.csv'
Traceback (most recent call last):
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/LOGistICS.py", line 30, in <module>
    import template_selector as ts
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/template_selector.py", line 5, in <module>
    df = pd.read_csv('template.csv', delimiter=',',header=0)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 317, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1729, in _make_engine
    self.handles = get_handle(
  File "/usr/lib/python3.10/site-packages/pandas/io/common.py", line 857, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'template.csv'
Traceback (most recent call last):
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/LOGistICS.py", line 30, in <module>
    import template_selector as ts
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/template_selector.py", line 5, in <module>
    df = pd.read_csv('template.csv', delimiter=',',header=0)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 317, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1729, in _make_engine
    self.handles = get_handle(
  File "/usr/lib/python3.10/site-packages/pandas/io/common.py", line 857, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'template.csv'
Traceback (most recent call last):
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/LOGistICS.py", line 30, in <module>
    import template_selector as ts
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/template_selector.py", line 5, in <module>
    df = pd.read_csv('template.csv', delimiter=',',header=0)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 317, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1729, in _make_engine
    self.handles = get_handle(
  File "/usr/lib/python3.10/site-packages/pandas/io/common.py", line 857, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: 'template.csv'
Traceback (most recent call last):
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/LOGistICS.py", line 30, in <module>
    import template_selector as ts
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/template_selector.py", line 5, in <module>
    df = pd.read_csv('./template.csv', delimiter=',',header=0)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 317, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1729, in _make_engine
    self.handles = get_handle(
  File "/usr/lib/python3.10/site-packages/pandas/io/common.py", line 857, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: './template.csv'
Traceback (most recent call last):
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/LOGistICS.py", line 30, in <module>
    import template_selector as ts
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/template_selector.py", line 5, in <module>
    df = pd.read_csv('./template.csv', delimiter=',',header=0)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 317, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1729, in _make_engine
    self.handles = get_handle(
  File "/usr/lib/python3.10/site-packages/pandas/io/common.py", line 857, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: './template.csv'
Traceback (most recent call last):
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/LOGistICS.py", line 30, in <module>
    import template_selector as ts
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/template_selector.py", line 5, in <module>
    df = pd.read_csv('./template.csv', delimiter=',',header=0)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 317, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1729, in _make_engine
    self.handles = get_handle(
  File "/usr/lib/python3.10/site-packages/pandas/io/common.py", line 857, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: './template.csv'
Traceback (most recent call last):
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/LOGistICS.py", line 30, in <module>
    import template_selector as ts
  File "/home/emanuele/Desktop/LOGistICS/LOGistICS/Snap/Modbus/template_selector.py", line 5, in <module>
    df = pd.read_csv('./template.csv', delimiter=',',header=0)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/util/_decorators.py", line 317, in wrapper
    return func(*args, **kwargs)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/usr/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1729, in _make_engine
    self.handles = get_handle(
  File "/usr/lib/python3.10/site-packages/pandas/io/common.py", line 857, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: './template.csv'
